sia/host/proofs.go

```diff
@@ -127,7 +127,7 @@ func (h *Host) createStorageProof(entry ContractEntry, heightForProof consensus.
 		err = errors.New("no record of that file")
 		return
 	}
-	fullname := h.hostDir + contractObligation.filename
+	fullname := filepath.Join(h.hostDir, contractObligation.filename)
```

Comment:
we should standardize on using `filepath.Join` instead of `path.Join`. They are almost identical, except that `filepath.Join` uses an OS-specific separator, while `path.Join` always uses `"/"`